jedi/settings.py

```diff
@@ -62,6 +63,11 @@
 The completion is by default case insensitive.
 """
 
+fuzzy_completion = False
+"""
+The completion is non-fuzzy by default.
+"""
+
 add_bracket_after_function = False
```

Comment:
I'm trying to get rid of the settings file more and more. It's a bad idea and params are better. Could you just use it as a param for `completions(fuzzy=False)`?

Also please rename completions for this and create this function:
```
def completions(self, **kwargs):
    return self._completions(**kwargs)

def _completions(self, fuzzy=False): ...
```

This makes it possible to use kwargs only arguments in Python 2 and it will be easy to migrate once Jedi is Python 3 only.